app/models/role.rb

```diff
@@ -171,6 +182,7 @@ def clone(role_params = {})
   end
 
   def locked?
+    return false if self.class.modify_locked
```

Comment:
I know the check is there, i was wondering if it made sense to move that here as well, so `locked?` will return `false` if ignore locking is used either on the instance or class level.